src/diff.jl

```diff
@@ -113,6 +113,25 @@ function expand_derivatives(O::Symbolic, simplify=false; occurances=nothing)
                     return expand_derivatives(operation(arg)(inner), simplify)
                 end
             end
+        elseif isa(operation(arg) , Integral)
+            if isa(operation(arg).domain , AbstractInterval)
+                domain = operation(arg).domain
+                a , b = DomainSets.endpoints(domain)
+                c = 0
+                inner_function = expand_derivatives_(arguments(arg)[1])
+                if isa(value(a), Term)
+                    t1 = replaceSym(operation(arg).x ,value(a) , inner_function )
+                    t2 = D(a)
+                    c -= t1*t2
+                elseif isa(value(b) , Term)
+                    t1 = replaceSym(operation(arg).x ,value(b) , inner_function )
```

Comment:
formatting needs to be cleaned up, we also never use camelCase function names. Good to stick to the package's style when you're contributing to it. Where is the `expand_derivatives_` function...?

Reply:
you could use `substitute` here.

Reply:
Yes, I infact removed `replaceSym`